docs/source/cuda-reference/kernel.rst

```diff
@@ -463,6 +463,51 @@ precision parts of the CUDA Toolkit documentation.
    ``cbrt`` and ``cbrtf`` in the C api. Supports float32, and float64 arguments
    only.
 
+16-bit Floating Point Intrinsics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Does this doc section need a small introduction?

Reply:
I've added a small introduction.

Reply:
Thanks